guide/build-config.md

```diff
@@ -92,8 +92,9 @@ builds:
 
   scmRevision: <scm revision>
 
-  environmentId: <id> # Either specify environmentId or systemImageId
-  systemImageId: <system image id> # Either specify environmentId or systemImageId
+  environmentId: <id> # Either specify environmentId, environmentName or systemImageId
```

Comment:
I would modify this section:
# Either specify environmentId, environmentName or systemImageId. If more than one is specified, the priority will be: environmentId, environmentName, systemImageId
environmentId: <id> # The specific id of a build environment, e.g. `20`
systemImageId: <system image id> # The specific builder image name associated with a build environment, e.g. `builder-rhel-7-j8-mvn3.6.0:1.0.0`
environmentName: <env-name> # The name of the builder image, e.g. `OpenJDK 1.8; Mvn 3.6.0`. As this will pick the latest up-to-date version of the image, it's the *recommended option*
  systemImageId: <system image id> # Either specify environmentId, environmentName or systemImageId

Reply:
Good suggestion. Is this format ok?

Reply:
Argh, not sure how to format it as a yaml comment without being picked from Markdown as a heading. maybe we can move this under the "#### environmentId v/s environmentName v/s systemImageId" heading, in bold?